src/haz3lcore/statics/Info.re

```diff
@@ -95,7 +97,9 @@ type ok_common =
   | Ana(ok_ana);
 
 [@deriving (show({with_path: false}), sexp, yojson)]
-type ok_exp = ok_common;
+type ok_exp =
+  | AnaDeferralConsistent(Typ.t)
```

Comment:
Can you remind me why we need this additional ok status? I don't think we need if for status_exp because we already know by construction in maketerm that every deferral inside a partial ap will be a Deferral(Used))